plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/totals/coupon/style.scss

```diff
@@ -22,7 +21,6 @@
 	}
 
 	.wc-block-components-totals-coupon__button {
-		height: em(48px);
```

Comment:
need height to automatically adjust based on the height of the inout next to the button, this prevents that.

Reply:
With this change, you just fixed #46627. 😁 I've linked the issue in the PR description so that it automatically gets closed when this PR gets merged.

Reply:
Oh that's great! Love closing issues by accident!